wordle_solver/wordle_solver.py

```diff
@@ -0,0 +1,66 @@
+from enum import Enum
+from string import ascii_lowercase
+from itertools import groupby
+
+from .word_initializers.english_words_initilalizer import EnglishWords
```

Comment:
Possibly better version (auto-corrected by PyCharm):
from word_initializers.english_words_initilalizer import EnglishWords
PyCharm warns me: (Not an error)
ImportError: attempted relative import with no known parent package